src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -138,13 +138,20 @@
   @#   The domain name of the API remote host.
   @# @@param port [Integer]
   @#   The port on which to connect to the remote host.
+  # TODO(landrito): add documentation specifying that the channel updater_proc and keyfile
```

Comment:
For trackability, would prefer an issue to a comment for this sort of thing, unless you're planning to fix this immediately in a follow-up PR.